package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tableland/evm",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
This is all backward compatable and doesn't really have any new features.  We are just adding to whats published